lib/modules/datasource/deb/readme.md

```diff
@@ -55,3 +58,50 @@ RUN apt-get update && \
 ```
 
 When the apt package for `gcc` is updated, Renovate updates the environment variable.
+
+```json title="Overwrite deb registryUrl via packageRule"
+{
+  "packageRules": [
+    {
+      "matchDatasources": ["deb"],
+      "matchPackageNames": ["gcc-11"],
+      "registryUrls": [
+        "https://deb.debian.org/debian?suite=stable&components=main,contrib,non-free&binaryArch=amd64"
+      ]
+    }
+  ]
+}
+```
+
+## Artifactory
+
+This datasource can also be used with Artifactory.
+The supported repository types are:
+
+- virtual
+- local
+- remote
+
+If you are using Artifactory, you can use the `deb` datasource with following `registryUrl` format:
+
+```
```

Comment:
Can you add this bit after the three backticks for the codeblock?

` title="Example of valid registryUrl format"`